packages/fx-core/templates/plugins/resource/hosting/bicep/webapp/config.template.bicep

```diff
@@ -0,0 +1,12 @@
+// Get existing app settings for merge
+var botCurrentAppSettings = list('${provisionOutputs.botOutput.value.botWebAppResourceId}/config/appsettings', '2021-02-01').properties
```

Comment:
This is a little tricky that we put bot concept in this bicep file. @hund030 what do you think?